math/dlib/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake  1.1
+PortGroup           cxx11  1.1
+
+github.setup        davisking dlib v19.17
```

Comment:
We don't want the version to start with a "v". So use this instead:

`github.setup        davisking dlib 19.17 v`